packages/ffe-core-react/src/index.js

```diff
@@ -7,3 +7,4 @@ export { default as Paragraph } from './typography/Paragraph';
 export { default as PreformattedText } from './typography/PreformattedText';
 export { default as SmallText } from './typography/SmallText';
 export { default as StrongText } from './typography/StrongText';
+export { default as Wave } from './wave/wave';
```

Comment:
Filenavnet burde begynne med en storbokstav fordi det er en klasse/komponent. Også er det ikke nødvendig å ha en egen mappe for den.
```js
exports { default as Wave } from './Wave';
```